stdlib.sh

```diff
@@ -654,12 +840,443 @@ function __STDLIB_API_1_info() { # {{{
 # This function may be overridden
 #
 function __STDLIB_API_1_debug() { # {{{
-	(( std_DEBUG )) && __STDLIB_API_1_std::log >&2 "DEBUG: " "${*:-Unspecified message}"
+	local message="${1:-Unspecified message}"
+	local -i level=${2:-$STDLIB_DEBUG_NORMAL}
```

Comment:
I think this is wrong: Previously, the `"${*}"` swallowed all arguments, whereas this change only allows the message in the first parameter, so `debug a b c` would now be interpreted as `debug a 0`!
Variadic (e.g. variable-number-of-parameter) functions likely need and flags/status variables first, c.f. `printf()` or even the `stdlib` `output` function.